cohorts-router.js

```diff
@@ -0,0 +1,92 @@
+const knex = require('knex');
+const knexConfig = require('./knexfile.js');
+const db = knex(knexConfig.development);
```

Comment:
this can be shortened up _even more_ by exporting from your knexfile.js `knex(knexConfig.development)` instead of just `knexConfig` -- then you only have to say `const db = require('./knexfile.js')` without importing knex here, or specifically passing in the knexConfig.development object. completely optional, but it's helpful in keeping your head wrapped around how all these exports/requires can be used to your advantage